src/lib.rs

```diff
@@ -1253,8 +1252,14 @@ pub extern "C" fn zendoo_batch_verify_proofs_by_id(
     if prioritize { zendoo_pause_low_priority_threads(); }
 
     // Execute batch verification of the proofs with specified id
-    let result = get_batch_verifier_thread_pool(prioritize)
-        .install(|| rs_batch_verifier.batch_verify_subset(rs_ids_list.to_vec(), &mut OsRng::default()));
+    let result = match get_batch_verifier_thread_pool(prioritize) {
+        Ok(pool) => pool.install(|| rs_batch_verifier.batch_verify_all(&mut OsRng::default())),
```

Comment:
Here you should call "batch_verify_subset"

Reply:
Sorry, done